en/backup-to-s3.md

```diff
@@ -63,6 +74,17 @@ For the current S3-compatible storage types, Ceph and Amazon S3 work normally as
 >     - --ignore-checksum
 > ```
 
+> **Note**
+>
+> This section lists multiple methods. Only follow the method that matches your situation.
```

Comment:
```suggestion
> This section lists multiple backup methods. Only follow the method that matches your situation.
```

Reply:
I would not consider these to be backup methods. I think they are "storage authentication methods".